src/pivot-table/components/shared-styles.ts

```diff
@@ -94,8 +94,8 @@ export const getTotalCellDividerStyle = ({
 
 export const getLeftGridStyles = (leftGridWidth: number): React.CSSProperties => ({
   width: leftGridWidth,
-  overflow: "scroll",
   position: "relative",
   display: "flex",
   flexDirection: "column",
+  overflow: "visible",
```

Comment:
I guess this was for testing purpose?

Reply:
there is actually no need for those inner (wrapper) divs now (and consequently this style as well) since I changed the children of `<StickyContainer />` to `ReactNode` -> now it can accept multiple children. 

this will be removed